yarn.lock

```diff
@@ -2363,7 +2363,7 @@
   resolved "https://registry.yarnpkg.com/@types/json5/-/json5-0.0.29.tgz#ee28707ae94e11d2b827bcbe5270bcea7f3e71ee"
   integrity sha1-7ihweulOEdK4J7y+UnC86n8+ce4=
 
-"@types/jwt-simple@^0.5.33":
+"@types/jwt-simple@0.5.33":
```

Comment:
not sure why this is changed

Reply:
This PR https://github.com/ChainSafe/lodestar/pull/3777 did not update the yarn.lock after requiring a different version. Running yarn causes this diff